cfme/tests/v2v/test_v2v_migrations.py

```diff
@@ -192,21 +192,7 @@ def _cleanup():
         soft_assert(src_vm.mac_address == migrated_vm.mac_address)
 
 
-@pytest.mark.parametrize(
-    "source_type, dest_type, template_type",
-    [
-        ["VM Network", "ovirtmgmt", Templates.DUAL_NETWORK_TEMPLATE],
-        ["DPortGroup", "Storage - VLAN 33", Templates.DUAL_NETWORK_TEMPLATE]
-    ],
-)
-@pytest.mark.uncollectif(
-    lambda source_provider, source_type:
-    source_provider.one_of(VMwareProvider) and source_provider.version != 6.5 and
-    "DPortGroup" in source_type,
-    reason='Dual network of DPortGroup only supported on source provider version 6.5'
-)
 def test_dual_nics_migration(request, appliance, provider,
```

Comment:
It will still not apply for Vmware67-ims and vmware6.0 I think . Please check , else you will have to add this uncollect code again

Reply:
@sshveta  DPortGroup is now available in Vsphere67-ims , So we are good to go with it and AFAIK we are not executing our suits on vsphere6.0, So we are good to remove this uncollected.